sms2jwplayer/genupdatejob.py

```diff
@@ -52,6 +52,41 @@ def main(opts):
         process_videos(fobj, items, videos)
 
 
+def convert_acl(visibility, acl):
+    """
+    Converts the old visibility and acl fields of :py:`~csv.MediaItem` to the new ACL scheme as
+    defined here: :py:`~csv.MediaItem.acl`.
+
+    :param visibility: :py:`~csv.MediaItem.visibility`
+    :param acl: :py:`~csv.MediaItem.acl` (list)
+    :return: the converted ACL
+    """
+    new_acl = []
+
+    def has_acl():
+        """Captures the case where the ACL [''] means no ACL """
+        return len(acl) != 1 or acl[0] if acl else False
+
+    if visibility == 'world-overrule' or (visibility == 'world' and not has_acl()):
+        new_acl.append('WORLD')
+    elif visibility == 'cam-overrule' or (visibility == 'cam' and not has_acl()):
+        new_acl.append('CAM')
+
+    if has_acl():
+        for ace in acl:
+            if ace.isdigit():
+                new_acl.append(f'GROUP_{ace}')
+            elif ace.upper() == ace:
```

Comment:
Media id 1940214 has ACL of "AC800" which is a CRSid (I think) but would be treated here as an inst id.

I think if it matches the regexp ``^[0-9]+$`` it is a group, ``^[A-Za-z]+[0-9]+$`` it is a CRSid and ``^[A-Za-z]+$``, it is a instid. If it matches none of those, I think an error should be logged.